src/styles/utils/FontUtils/fontFamily/multiFontFamily.ts

```diff
@@ -8,7 +8,7 @@ import type FontFamilyStyles from './types';
 const fontFamily: FontFamilyStyles = {
     EXP_NEUE_ITALIC: 'ExpensifyNeue-Italic, Segoe UI Emoji, Noto Color Emoji',
     EXP_NEUE_BOLD: multiBold,
-    EXP_NEUE: 'ExpensifyNeue-Regular, Segoe UI Emoji, Noto Color Emoji',
+    EXP_NEUE: 'ExpensifyNeue-Regular, Segoe UI Emoji',
```

Comment:
Segoe UI Emoji will be applied first, the order is important.

Reply:
For emoji, we only use EXP_NEUE.

if we keep Noto Color Emoji here, we can't have the needed order, Noto Color Emoji should be after ExpensifyNeue-Regular.

Reply:
I don't know, but what I am sure about is:
- it was not working, it was there for nothing
- On windows, Segoe UI Emoji will be used because it's a system font.

For now, it's still there, eeven if it has a specific use case, we are keeping it. We here just removing Segoe UI Emoji (it's only a Windows OS default font)